tests/test_octodns_manager.py

```diff
@@ -79,6 +79,13 @@ def test_missing_source(self):
             )
         self.assertTrue('missing sources' in str(ctx.exception))
 
+    def test_missing_zone(self):
+        with self.assertRaises(ManagerException) as ctx:
+            Manager(get_config_filename('dynamic-config.yaml')).sync(
+                ['missing.zones.']
+            )
+        self.assertTrue('Requested zone:' in str(ctx.exception))
```

Comment:
Didn't notice the `:`

```suggestion
        self.assertTrue('Requested zone ' in str(ctx.exception))
```